src/index.html

```diff
@@ -6,13 +6,72 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="styles/index.scss"
```

Comment:
The stylesheet link uses a '.scss' file. Browsers do not support SCSS files directly. You need to compile the SCSS file to a CSS file and link the resulting '.css' file instead.